devops/cloudbuild-pr.yaml

```diff
@@ -16,10 +16,12 @@ steps:
   - name: node:16.14.2
     entrypoint: npm
     env:
+      - 'VUE_APP_ADDRESS_COMPLETE_KEY=$_VUE_APP_ADDRESS_COMPLETE_KEY'
```

Comment:
So can these be changed to VITE_xxx ? Should they? Will this work with vaults.env and .env?

Reply:
OK for now but I'd like to see a new ticket to clean this up (and cc Patrick on it since it will need his help). Thx.

Reply:
New ticket: https://github.com/bcgov/entity/issues/17334